src/main/java/view/Menu.java

```diff
@@ -0,0 +1,30 @@
+package view;
+
+import java.util.Arrays;
+import java.util.Objects;
+
+public enum Menu {
+
+    CHECK_ATTENDANCE("1"),
+    UPDATE_ATTENDANCE("2"),
+    CREW_ATTENDANCE_RECORD("3"),
+    PENALTY_RECORD("4"),
+    QUIT("Q");
+
+    private final String menuSelector;
+
+    Menu(String menuSelector) {
+        this.menuSelector = menuSelector;
+    }
+
+    public static Menu findBySelector(String inputMenuSelector) {
+        return Arrays.stream(Menu.values())
+                .filter(menu -> Objects.equals(menu.menuSelector, inputMenuSelector))
+                .findFirst()
+                .orElseThrow(() -> new IllegalArgumentException("[ERROR] 메뉴를 찾을 수 없습니다"));
+    }
+
+    public static boolean isQuit(Menu menu) {
+        return menu == QUIT;
+    }
```

Comment:
static 메서드가 아닌 일반 메서드로 선언할 수 있을 것 같아요.
```suggestion
    public boolean isQuit() {
        return this == QUIT;
    }
```

Reply:
아 그럴수 있네요! 필요치 않는 static 메소드를 사용했던 것 같습니다
좀더 고민해보고 사용해볼게요!
7484ecf74a02891aac31ab583f7c5ff480586ced